cartography/intel/aws/ec2/internet_gateways.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+
+from .util import get_botocore_config
+from cartography.util import aws_handle_regions
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_internet_gateways(boto3_session: boto3.session.Session, region: str) -> List[Dict]:
+    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
+    return client.describe_internet_gateways()['InternetGateways']
+
+
+@timeit
+def load_internet_gateways(
+    neo4j_session: neo4j.Session, data: List[Dict], region: str,
+    current_aws_account_id: str, update_tag: int,
+) -> None:
+    logger.debug("Loading %d Internet Gateways in %s.", len(data), region)
+    query = """
+            MERGE (ig:AWSInternetGateway{id: {InternetGatewayId}})
```

Comment:
It might be worth rewriting this to use `UNWIND` to something like this:  https://github.com/lyft/cartography/blob/047a6bd1820e846e70bda4c6ddc18027da39e20a/cartography/intel/aws/elasticache.py#L53

This way, the assets get loaded to the graph in one large transaction instead of N transactions, which  can be [significantly faster](https://achantavy.github.io/cartography/performance/cypher/neo4j/2020/07/19/loading-7m-items-to-neo4j-with-and-without-unwind.html).

Reply:
Thanks for sharing the informative blog post!